mlflow/pyspark/ml/__init__.py

```diff
@@ -0,0 +1,134 @@
+import json
+
+import mlflow
+from mlflow.utils import chunk_dict, truncate_dict
+from mlflow.utils.annotations import experimental
+from mlflow.utils.autologging_utils import (
+    autologging_integration,
+    safe_patch,
+    try_mlflow_log,
+)
+
+FLAVOR_NAME = "pyspark_ml"
+
+
+def _get_estimator_info_tags(estimator):
+    """
+    :return: A dictionary of MLflow run tag keys and values
+             describing the specified estimator.
+    """
+    return {
+        "estimator_name": estimator.__class__.__name__,
+        "estimator_class": (estimator.__class__.__module__ + "." + estimator.__class__.__name__),
+    }
+
+
+def _get_estimator_param_map(estimator):
+    param_map = {param.name: estimator.getOrDefault(param)
+                for param in estimator.params if estimator.isDefined(param)}
+    # TODO:
+    #  handle special case: OneVsRest.classifier,CrossValidator.estimator,
+    #  CrossValidator.estimatorParamMaps
+    json_param_map = {name: json.dumps(param_map[name]) for name in param_map}
```

Comment:
Why do we need jsonize values here?

Reply:
Spark params may get value type like boolean, int

Reply:
I think `truncate_dict` automatically stringifies both keys and values.